src/Modules/Build.lua

```diff
@@ -367,14 +367,17 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
 		{ stat = "BleedDPS", label = "Bleed DPS", fmt = ".1f", compPercent = true, warnFunc = function(v) return v >= data.misc.DotDpsCap and "Bleed DPS exceeds in game limit" end },
 		{ stat = "CorruptingBloodDPS", label = "Corrupting Blood DPS", fmt = ".1f", compPercent = true, warnFunc = function(v,o) return v >= data.misc.DotDpsCap and "Corrupting Blood DPS exceeds in game limit" end },
 		{ stat = "BleedDamage", label = "Total Damage per Bleed", fmt = ".1f", compPercent = true, flag = "showAverage" },
+		{ stat = "BleedDuration", label = "Bleed duration", fmt = ".1f", compPercent = true},
```

Comment:
Would be nice to add "s" to the formatting here to indicate that it's in seconds
`{ stat = "BleedDuration", label = "Bleed duration", fmt = ".1fs", compPercent = true},`